src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.topology(KnownTopology.Samba)
+def test_sss_overrides__overriding_username_and_posix_attributes(client: Client, provider: GenericProvider):
+    """
+    :title: Locally overriding the name and POSIX attributes of a user
+    :setup:
+        1. Create POSIX user "user1" with standard POSIX attributes defined
+        2. Start SSSD
+    :steps:
+        1. Create local override for "user1" and name it "o-user1"
+        2: Restart SSSD, this is necessary to enable local overrides
```

Comment:
fix numbering typo from 2: to 2.

This typo is in all of the tests below and is throwing an error for the ci system pipelines.

Reply:
Found another error as well. One test case had two step 3s, nice catch.